iron-doc-behavior.html

```diff
@@ -6,11 +6,13 @@
 
       properties: {
         /**
-         * The [Hydrolysis](https://github.com/PolymerLabs/hydrolysis)-generated
+         * The [Polymer Analyzer](https://github.com/Polymer/polymer-analyzer)-generated
          * element descriptor to display details for.
          *
          * Alternatively, the element descriptor can be provided as JSON via the text content
          * of this element.
+         *
+         * @type {hydrolysis.ElementDescriptor}
```

Comment:
Copy pasted from the other file. This whole project needs a sweep through for a bunch of style things so I'm being pretty lazy.